### ISSUE TITLE
[5] TraitBank - show/hide curation tool is not working

### ISSUE THREAD

**Issue by jhammock** [EOL/tramea#96]
The tool is not working and not displayed.

This can wait for Tramea if needed.

**Comment by menna-hedaia** [comment]
Done. The solution is in the following branch 
https://github.com/EOL/eol/commit/6b50b7140cc10ab4d3b7688dc04fa547e92fb832
Thanks

**Comment by JRice** [comment]
A bit hard to follow the diff, but I'm willing to have this merged to master in order to see it working. Could you do that, please?

**Comment by menna-hedaia** [comment]
Code modified in the following branch 
https://github.com/EOL/eol/commit/e985b088fdc566b3cd6f36ca6056c484919961c1
and changes merged to master
